edi_sale_utm_oca/models/edi_exchange_type.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class EDIExchangeType(models.Model):
+    _inherit = "edi.exchange.type"
+
+    edi_source_id = fields.Many2one(
```

Comment:
Sorry but this is wrong. Ping @nilshamerlinck

Reply:
what if someone wants a different utm.source per exchange type?